pkg/metrics/metrics.go

```diff
@@ -141,6 +142,14 @@ var (
 		},
 	)
 
+	catalogSourceState = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "catalogSource_state",
```

Comment:
```suggestion
			Name: "catalogsource_unready",
```

Reply:
:+1: I was assuming here that my other suggestion about changing semantics was accepted.